src/tests/Microsoft.PowerFx.Core.Tests.Shared/RecursiveRecordTypeTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.PowerFx.Core.Texl.Builtins;
+using Microsoft.PowerFx.Syntax;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class RecursiveTypeTest : PowerFxTest
+    {
+        [Theory]
+        [InlineData("Collect(TableLoop, First(TableLoop), First(TableLoop))")]
+        public void NoStackoverflowTest(string expression)
```

Comment:
nit - is there an existing file this would below in?

Reply:
I didn't find a good fit. Let me know if you have a suggestion and I can move it later on.